system/HTTP/RequestTrait.php

```diff
@@ -66,6 +66,11 @@ public function getIPAddress(): string
         // @phpstan-ignore-next-line
         $proxyIPs = $this->proxyIPs ?? config(App::class)->proxyIPs;
         // @phpstan-ignore-next-line
+
+        // Workaround for old Config\App file. App::$proxyIPs may be empty string.
+        if ($proxyIPs === '') {
+            $proxyIPs = [];
+        }
```

Comment:
I think this can be simplified to a long ternary.
```suggestion
        $proxyIPs = $this->proxyIPs ?: config(App::class)->proxyIPs ?: [];
```

Reply:
It is shorter, but the current code is easier to understand why it should like that.
And the `if` statement will be removed soon in 4.4.

Reply:
Fair enough.